qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -88,7 +90,10 @@ public static void filterDistros() {
 
     @Before
     public void setupTest() throws IOException {
-        installation = runContainer(distribution(), builder().envVars(Map.of("ingest.geoip.downloader.enabled", "false")));
+        installation = runContainer(
+            distribution(),
+            builder().envVars(Map.of("ingest.geoip.downloader.enabled", "false", "ELASTIC_PASSWORD", PASSWORD))
```

Comment:
I would suggest we randomly disable Security (set `xpack.security.enabled` to `false`).

I think these tests are the only place where we can reliably assert the security "status" of a distribution, although until now these tests were Security agnostic.

Reply:
I don't think we need to randomize, moving forward we want security to be enabled  eitherway. I'll add a test case checking that we can make calls to the APIs without credentials when security is disabled, WDYT?

Reply:
That works for me.